app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express")
+const {getAllCategories} = require("./controllers/categories.controller.js")
+
+const app = express();
+
+app.use(express.json())
```

Comment:
This one is not needed at this point. In the future only include code that is being used.